src/tools/kata-ctl/src/args.rs

```diff
@@ -12,6 +12,35 @@ use thiserror::Error;
 pub struct KataCtlCli {
     #[clap(subcommand)]
     pub command: Commands,
+    #[clap(short, long, value_enum)]
+    /// Sets the minimum log level required for log messages to be displayed. Default is 'info'
+    pub log_level: Option<LevelWrapper>,
```

Comment:
Why not just use Option<String> and implement a branch on that string, this struct seems unnecessary.

Reply:
Thanks for the suggestion - I tried this initially and couldn't get it to work. Finally figured it out and it's smaller, better code with no extra struct using clap's `value_parser` feature.